src/Service/ServiceUtils.cpp

```diff
@@ -135,8 +144,17 @@ std::optional<Jiffies> GetCumulativeCpuTimeFromProcess(pid_t pid) {
   return Jiffies{utime + stime};
 }
 
-std::optional<TotalCpuTime> GetCumulativeTotalCpuTime() {
-  std::ifstream stat_stream{"/proc/stat"};
+std::optional<TotalCpuTime> GetCumulativeTotalCpuTime() noexcept {
+  ErrorMessageOr<std::string> stat_content = orbit_base::ReadFileToString("/proc/stat");
+  if (!stat_content) {
+    ERROR("%s", stat_content.error().message());
+    return {};
```

Comment:
```suggestion
    return std::nullopt;
```
For consistency with `GetCumulativeCpuTimeFromProcess` and the next if.

Reply:
done. also changed in other places.